src/css/widget.css

```diff
@@ -0,0 +1,89 @@
+.table {
```

Comment:
? ga eens scss gebruiken

Reply:
+ wat joeri al zegt, dit is echt veeeel te veel code. kan je niet gewoon de bestaande tabellen hergebruiken?